package/gluon-client-bridge/luasrc/lib/gluon/upgrade/300-gluon-client-bridge-network

```diff
@@ -40,31 +40,41 @@ uci:section('network', 'interface', 'client', {
 
 uci:save('network')
 
-
+-- TODO: remove this line and the next in 2019. Firewall zones have been renamed in 2017.
 uci:delete('firewall', 'client')
-uci:section('firewall', 'zone', 'client', {
-	name = 'client',
+
+uci:section('firewall', 'zone', 'drop', {
+	name = 'drop',
 	network = {'client'},
 	input = 'DROP',
 	output = 'DROP',
 	forward = 'DROP',
 })
 
-uci:save('firewall')
+uci:section('firewall', 'zone', 'local_client', {
+	name = 'local_client',
+	network = {'local_node'},
+	input = 'DROP',
+	output = 'ACCEPT',
+	forward = 'DROP',
```

Comment:
These should also use REJECT.

Reply:
done

Reply:
well... should the drop zone be renamed if it is actually doing reject?

Reply:
The drop zone should still use DROP, it should look to the network as if a drop interface doesn't exist.

Reply:
fine. 👍